.github/workflows/release.yml

```diff
@@ -104,12 +104,19 @@ jobs:
           git config --global --add safe.directory "$GITHUB_WORKSPACE"
           git describe --always --tags ${{ github.sha }}
           ./tools/release.sh
+      run: |
+          # once the build is over, we want to generate a Debian package
+          ./tools/packaging/generate_debian_package.sh build-opt/dragonfly-x86_64
+      run: |
+          # then place all artifacts at the same location
+          rm -rf results-artifacts && mkdir -p results-artifacts
```

Comment:
do you actually need `rm -rf results-artifacts` ? I think we can assume a clean workspace in CI :)

Reply:
yes it was little bit conservative, this is why I mark this WIP I want to see, how this works